packages/connect-node-test/src/crosstest/cancel_after_begin.spec.ts

```diff
@@ -28,38 +28,44 @@ describe("cancel_after_begin", function () {
     }
   }
 
-  servers.describeTransports((transport) => {
-    const servers = createTestServers();
-    beforeAll(async () => await servers.start());
-    it("with promise client", async function () {
-      const client = createPromiseClient(TestService, transport());
-      const controller = new AbortController();
-      async function* input() {
-        yield {
-          payload: {
-            body: new Uint8Array(),
-            type: PayloadType.COMPRESSABLE,
-          },
-        };
-        await new Promise((resolve) => setTimeout(resolve, 1));
-        controller.abort();
-        yield {
-          payload: {
-            body: new Uint8Array(),
-            type: PayloadType.COMPRESSABLE,
-          },
-        };
-      }
-      try {
-        await client.streamingInputCall(input(), {
-          signal: controller.signal,
-        });
-        fail("expected error");
-      } catch (e) {
-        expectError(e);
-      }
-    });
-  });
+  servers.describeTransportsExcluding(
+    [
+      "@bufbuild/connect (ConnectRouter, binary)",
+      "@bufbuild/connect (ConnectRouter, JSON)",
```

Comment:
I would expect this test case to work with the router transport too. How is it failing?

Reply:
it fails with:

```
Failures:
1) cancel_after_begin @bufbuild/connect (ConnectRouter, JSON) with promise client
  Message:
    Failed: expected error
  Stack:
        at <Jasmine>
        at UserContext.<anonymous> (file:///Users/dimitri/src/github.com/bufbuild/connect-es/packages/connect-node-test/dist/esm/crosstest/cancel_after_begin.spec.js:70:17)
```

Reply:
Okay, we have to fix this, but not here.